internal/service/ecs/service.go

```diff
@@ -90,6 +90,31 @@ func ResourceService() *schema.Resource {
 					},
 				},
 			},
+			"alarms": {
+				Type:             schema.TypeList,
+				Optional:         true,
+				MaxItems:         1,
```

Comment:
Where that does restriction come from? The API never mentions any upper limit, and definitely does not mention that only one item is allowed.